docs/source/spec/core/stream-traits.rst

```diff
@@ -440,8 +349,12 @@ service, followed by the events sent in the payload of the HTTP message.
             room: String,
```

Comment:
This is my bad, but this needs to be marked as `@required` too :)

This also needs to change in the initial response example